src/components/field/svg/fields/FieldDimensions.tsx

```diff
@@ -0,0 +1,3 @@
+// Auto-generated by update_field_dimensions.py
+export const FIELD_LENGTH = 16.5811;
```

Comment:
These should be used for the field keep in constraint

Reply:
The keep-in constraint will derive its values from this file. That's part of the near-future PR mentioned above.